uint16[] indices # table of output indices which states are requested
---
bool[] states